src/inspector_socket_server.cc

```diff
@@ -82,10 +82,10 @@ void PrintDebuggerReadyMessage(const std::string& host,
     return;
   }
   fprintf(out,
-          "Debugger listening on port %d.\n"
+          "Debugger listening on %s:%d.\n"
```

Comment:
might be best to defer to #11207 for changes to the hint text, although I agree we should include the host.